Source/Core/Core/PowerPC/Jit64/Jit.cpp

```diff
@@ -473,7 +473,7 @@ static void ImHere()
     if (!f)
       f.Open("log64.txt", "w");
 
-    fprintf(f.GetHandle(), "%08x\n", PC);
+    f.WriteString(fmt::format("{0:08x}", PC));
```

Comment:
Did you intentionally get rid of the trailing `\n`?

Reply:
Nope, thanks!